docs/book/src/plugins/deploy-image-plugin-v1-alpha.md

```diff
@@ -0,0 +1,46 @@
+# Deploy Image Plugin (deploy-image/v1-alpha)
+
+The deploy-image plugin allows you to create [controllers][controller-runtime] and custom resources which will deploy and manage an image on the cluster following
+the guidelines and best practices. It abstracts the complexities to achieve this goal and still giving the possibility of users improve and customize their projects
+according to their requirements.
+
+<aside class="note">
+<h1>Examples</h1>
+
+See the project "project-v3-with-deploy-image" under the [testdata][testdata] directory of the Kubebuilder project to check an example of the scaffold produced with this plugin.
+
+</aside>
+
+## When to use it ?
+
+- If you are getting started with the [Operator pattern](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/), Kubebuilder and [controller-runtime](https://github.com/kubernetes-sigs/controller-runtime) this plugin can help you
+- If you are looking to Deploy and Manage an image (Operand) using [Operator pattern](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) and the tool the plugin will create an API/controller to be reconciled and achieve this goal
+- If you are looking for examples and recommendations you might want to check out the scaffold done by this plugin
+
+## How to use it ?
+
+After you create a new project with `kubebuilder init` you can create APIs informing this plugin. Ensure that you followed up the [quick start](https://book.kubebuilder.io/quick-start.html) before trying to use it.
+
+Then, by using this plugin you can [create APIs][https://book.kubebuilder.io/cronjob-tutorial/gvks.html] informing the image (Operand) that you would like to deploy on the cluster. Note that you can optionally inform what is the command that should be used to initialize this container via the flag `--image-command` as the port that should be used with `--image-port`, i.e:
+
+```sh
+kubebuilder create api --group example.com --version v1alpha1 --kind Memcached --image=memcached:1.6.15-alpine --image-container-command="memcached -m=64 modern -v" --image-container-port="11211" --plugins="deploy-image/v1-alpha"
+```
+
+## Subcommands
+
+The deploy-image plugin implements the following subcommands:
+
+- create api (`$ kubebuilder create api [OPTIONS]`)
+
+## Affected files
+
+By calling create api with this plugin in additionally to what is created and updated by default we have the following customizations on top:
+
+- `controllers/*_controller.go`
+- `controllers/*_suite_test.go`
+- `api/*_types.go` (scaffold the specs for the new api)

```

Comment:
It should be `api/{version}/*_types.go`. And it will be `apis/{group}/{version}/*_types.go` when multigroup is enabled.

Reply:
Note that this is not fixed yet. It should be `api/<version>/*_types.go` instead of `api/*_types.go`  :)